prov/rxm/src/rxm_conn.c

```diff
@@ -184,8 +188,7 @@ ssize_t rxm_get_conn(struct rxm_ep *rxm_ep, fi_addr_t addr,
 	assert(rxm_ep->util_ep.tx_cq);
 	handle = rxm_cmap_acquire_handle(rxm_ep->cmap, addr);
 	if (!handle) {
-		ret = rxm_cmap_alloc_handle(rxm_ep->cmap, addr,
-					    RXM_CMAP_IDLE, &handle);
+		ret = rxm_cmap_alloc_handle(rxm_ep->cmap, addr, &handle);
```

Comment:
> It would be easier to check the changes if this and similar changes (rxm_cma_init_handle) were separated out and applied before adding the rx_state.

I'll do this and refresh the PR, thanks.